.aws/deploy/support-task-definition.staging.json

```diff
@@ -0,0 +1,311 @@
+          "valueFrom": "STAGING_SITE_LAUNCH_DYNAMO_DB_TABLE_NAME"
+        },
+        {
+          "name": "SITE_LAUNCH_FORM_KEY",
+          "valueFrom": "STAGING_SITE_LAUNCH_FORM_KEY"
+        },
+        {
+          "name": "SITE_PASSWORD_SECRET_KEY",
+          "valueFrom": "STAGING_SITE_PASSWORD_SECRET_KEY"
+        },
+        {
+          "name": "STAGING_SSH_PUBLIC_KEY",
+          "valueFrom": "STAGING_SSH_PUBLIC_KEY"
+        },
+        {
+          "name": "STAGING_SSH_PRIVATE_KEY",
+          "valueFrom": "STAGING_SSH_PRIVATE_KEY"
+        },
+        {
+          "name": "STEP_FUNCTIONS_ARN",
+          "valueFrom": "STAGING_STEP_FUNCTIONS_ARN"
+        },
+        {
+          "name": "SYSTEM_GITHUB_TOKEN",
+          "valueFrom": "STAGING_SYSTEM_GITHUB_TOKEN"
+        },
+        {
+          "name": "UPTIME_ROBOT_API_KEY",
+          "valueFrom": "STAGING_UPTIME_ROBOT_API_KEY"
+        },
+        {
+          "name": "SITE_CHECKER_FORM_KEY",
+          "valueFrom": "STAGING_SITE_CHECKER_FORM_KEY"
+        },
+        {
+          "name": "SITE_AUDIT_LOGS_FORM_KEY",
+          "valueFrom": "STAGING_SITE_AUDIT_LOGS_FORM_KEY"
+        }
+      ],
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "/aws/elasticbeanstalk/cms-backend-staging-node18/var/log/web.stdout.log",
+          "awslogs-region": "ap-southeast-1",
+          "awslogs-stream-prefix": "ecs"
+        }
+      }
+    },
+    {
+      "name": "dd-agent",
+      "image": "public.ecr.aws/datadog/agent:latest",
+      "portMappings": [
+        {
+          "containerPort": 8126,
+          "hostPort": 8126,
+          "protocol": "tcp"
+        }
+      ],
+      "essential": true,
+      "environment": [
+        {
+          "name": "TZ",
+          "value": "Asia/Singapore"
+        },
+        {
+          "name": "DD_APM_NON_LOCAL_TRAFFIC",
+          "value": "true"
+        },
+        {
+          "name": "ECS_FARGATE",
+          "value": "true"
+        },
+        {
+          "name": "DD_APM_ENABLED",
+          "value": "true"
+        },
+        {
+          "name": "DD_SITE",
+          "value": "datadoghq.com"
+        },
+        {
+          "name": "DD_ENV",
+          "value": "staging"
+        },
+        {
+          "name": "DD_SERVICE",
+          "value": "isomer"
+        },
+        {
+          "name": "DD_TAGS",
+          "value": "service:isomer"
+        },
+        {
+          "name": "DD_AGENT_MAJOR_VERSION",
+          "value": "7"
+        },
+        {
+          "name": "DD_LOGS_INJECTION",
+          "value": "true"
+        },
+        {
+          "name": "DD_TRACE_STARTUP_LOGS",
+          "value": "true"
+        },
+        {
+          "name": "DD_API_KEY",
+          "value": "<DD_API_KEY>"
+        }
+      ],
+      "dockerLabels": {
+        "com.datadoghq.tags.env": "staging",
+        "com.datadoghq.tags.service": "isomer",
```

Comment:
When service is renamed, this will need to change as well.

Reply:
updated, see above comment